### PR TITLE
[PPP-4747] Vulnerable Component: batik-1.19

### PR DESCRIPTION
**Pull request by SahithiKommagani6**
[PPP-4747] Vulnerable Component: batik-1.19

[PPP-4747]: https://hv-eng.atlassian.net/browse/PPP-4747?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 52s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: pom.xml [hunk 1]
```diff
@@ -192,6 +192,8 @@
     <simple-jndi.version>1.0.13</simple-jndi.version>
 
     <batik.version>1.16</batik.version>
```
[the remark this answers is not in the document]
Comment (krishnaalluhitachi) [reply pom.xml:194]: Yes..it should be 1.17

[the remark this answers is not in the document]
Comment (krishnaalluhitachi) [reply pom.xml:194]: @lucboudreau just connected with @SahithiKommagani6 and she clarified that there are other components like batik-util, batik-codec etc. which are still referring to **batik.version** property even the component is not batik-bridge or batik-css. So she didn't remove the propety and declared new properties as below to upgrade the batik-bridge & batik-css components.
   
    <batik-bridge.version>1.17</batik-bridge.version>
    <batik-css.version>1.17</batik-css.version>

[the remark this answers is not in the document]
Comment (peterrinehart) [reply pom.xml:194]: Do we need to upgrade all of the related components, or are we safe to only upgrade batik-bridge and batik-css?

[the remark this answers is not in the document]
Comment (krishnaalluhitachi) [reply pom.xml:194]: As per the JIRA task we are upgrading only batik-bridge and batik-css, I will initiate another CodeDX after the master merge to find if something needs to be upgraded still...

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 2s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 4s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 52s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 55s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 59s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 2s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Review by krishnaalluhitachi — APPROVED** [review]

**Review by cardosov — APPROVED** [review]
Merging by request from @krishnaalluhitachi.

**Pull request merged**